<launch>
  <arg name="pcap"  default=""/>
  <arg name="device_ip" default="192.168.1.201"/>
  <arg name="lidar_port"  default="2368"/>
  <arg name="gps_port"  default="10110"/>
  <arg name="scan_phase"  default="0"/>
  <arg name="model" default="Pandar40P"/>
  <arg name="frame_id" default="pandar"/>


  <node pkg="pandar_driver" exec="pandar_driver_node" name="pandar_driver" output="screen" >
    <param name="pcap" type="string" value="$(var pcap)"/>
    <param name="device_ip" type="string" value="$(var device_ip)"/>
    <param name="lidar_port"  type="int" value="$(var lidar_port)"/>
    <param name="gps_port"  type="int" value="$(var gps_port)"/>
    <param name="scan_phase"  type="double" value="$(var scan_phase)"/>
    <param name="model"  type="string" value="$(var model)"/>
    <param name="frame_id"  type="string" value="$(var frame_id)"/>
  </node>
</launch>
